solidity/ecdsa/hardhat.config.ts

```diff
@@ -87,23 +87,23 @@ const config: HardhatUserConfig = {
           testConfig.nonStakingAccountsCount +
           testConfig.stakingRolesCount * testConfig.operatorsCount,
       },
-      tags: ["allowStubs"],
       // we use higher gas price for tests to obtain more realistic results
       // for gas refund tests than when the default hardhat ~1 gwei gas price is
       // used
-      gasPrice: 200000000000, // 200 gwei,
+      gasPrice: 200000000000, // 200 gwei
       // Ignore contract size on deployment to hardhat network, to be able to
       // deploy stub contracts in tests.
       allowUnlimitedContractSize: process.env.TEST_USE_STUBS_ECDSA === "true",
+      tags: ["allowStubs"],
     },
     development: {
       url: "http://localhost:8545",
       chainId: 1101,
       tags: ["allowStubs"],
     },
-    ropsten: {
+    goerli: {
       url: process.env.CHAIN_API_URL || "",
-      chainId: 3,
+      chainId: 5,
       accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
         ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
```

Comment:
Do we want to make similar change to [this one](https://github.com/keep-network/keep-core/pull/3111) here?